test/operations.py

```diff
@@ -0,0 +1,61 @@
+import requests
+import json
+from typing import Optional, Tuple
+
+
+def create_index(endpoint: str, index_name: str, auth: Optional[Tuple[str, str]] = None, data: Optional[dict] = None):
+    response = requests.put(f'{endpoint}/{index_name}', auth=auth, verify=False)
+
+    return response
+
+
+def check_index(endpoint: str, index_name: str, auth: Optional[Tuple[str, str]] = None, data: Optional[dict] = None):
+    response = requests.get(f'{endpoint}/{index_name}', auth=auth, verify=False)
+
+    return response
+
+
+def delete_index(endpoint: str, index_name: str, auth: Optional[Tuple[str, str]] = None, data: Optional[dict] = None):
+    response = requests.delete(f'{endpoint}/{index_name}', auth=auth, verify=False)
+
+    return response
+
+
+def delete_document(endpoint: str, index_name: str, doc_id: str, auth: Optional[Tuple[str, str]] = None):
+    response = requests.delete(f'{endpoint}/{index_name}/_doc/{doc_id}', auth=auth, verify=False)
+
+    return response
+
+
+def create_document(endpoint: str, index_name: str, doc_id: str, auth: Optional[Tuple[str, str]] = None,
+                    data: Optional[dict] = None):
```

Comment:
Can you elaborate on why this function has `data` as well? I expected `data` to be the document that was being created, but it actually doesn't seem to be used at all.